### PR TITLE
Clean up resnet, remove batch norms

### PR DESCRIPTION
**Pull request by adamgayoso**

### CODE REVIEW

**Comment by adamgayoso** [comment]
I ran this on my local notebooks, results look more or less similar but training appears more stable

**Review by PierreBoyeau — APPROVED** [review]
This looks good to me. Just in case: do you know if there is an easy way to have both options (LayerNorm or BatchNorm)? I would assume not but I was wondering if you saw implementations in Jax with this.
